src/persist-client/src/impl/state.rs

```diff
@@ -201,6 +197,26 @@ where
         }
         self.since = since;
     }
+
+    fn push_batch(&mut self, keys: &[String], desc: &Description<T>) {
```

Comment:
this impl is just a placeholder. let's finish hashing out what specifically we'd like this to do